iis-workflows/pom.xml

```diff
@@ -56,20 +56,15 @@
         <oozie.package.file.name>oozie-package</oozie.package.file.name>
         <!-- notice: sandboxName is generated based on workflow.source.dir property -->
         <workflow.source.dir>src/test/resources/define/path/pointing/to/directory/holding/oozie_app</workflow.source.dir>
```

Comment:
Not exactly. Maven is flexible with properties. If he finds something that look like property placeholder (eg. `${workflow.source.dir}`) but property doesn't exist, then it would leave it as is and continue build.
In our case it would result in priming that will lead to success, but build will fail in next stages. Error will say something about circular reference in property definitions which don't point us directly to source problem.
When `workflow.source.dir` is provided in pom, then build will fail while priming. Giving us message that `src/test/resources/define/path/pointing/to/directory/holding/oozie_app` was not in classpath. So cause of build fail is easier to find.